OAT.Tests/CaptureTests.cs

```diff
@@ -0,0 +1,835 @@
+﻿using KellermanSoftware.CompareNetObjects;
+using Microsoft.CST.OAT.Utils;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.CST.OAT.Tests
+{
+    [TestClass]
+    public class CaptureTests
+    {
+        [ClassInitialize]
+        public static void ClassSetup(TestContext _)
+        {
+            Logger.SetupVerbose();
+            Strings.Setup();
+        }
+
+        private const string CorrectString = "MagicWord";
+
+        private TestObject testObjectTrueFalse = new TestObject()
+        {
+            StringField = CorrectString,
+            BoolField = false
+        };
+
+        [TestMethod]
+        public void TestEQCapture()
+        {
+            var RuleName = "Equals Capture";
+            var analyzer = new Analyzer();
+
+            var eqCaptureRule = new Rule(RuleName)
+            {
+                Clauses = new List<Clause>()
+                {
+                    new Clause(OPERATION.EQ, "StringField")
```

Comment:
Any reason why the OPERATION enum is all caps?

Reply:
Just stylistically chose enums to be all caps.

Reply:
See update.  I changed the Operation enum to be PascalCase.